moveit_core/constraint_samplers/src/constraint_sampler_tools.cpp

```diff
@@ -103,20 +106,20 @@ void constraint_samplers::visualizeDistribution(const ConstraintSamplerPtr& samp
     if (!sampler->sample(ks))
       continue;
     const Eigen::Vector3d& pos = ks.getGlobalLinkTransform(lm).translation();
-    visualization_msgs::Marker mk;
-    mk.header.stamp = ros::Time::now();
+    visualization_msgs::msg::Marker mk;
+    mk.header.stamp = rclcpp::Clock().now();
```

Comment:
We want RCL_ROS_TIME here not RCL_SYSTEM_TIME 

http://docs.ros2.org/latest/api/rcl/time_8h.html#a5c734f508ce06aec7974af10ad09d071a423b9ea49c5f5119c3b9b812c01e6bc9

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/23e2af614eab5f1ecc27f2299c40987d4200f183